holidays/countries/tuvalu.py

```diff
@@ -0,0 +1,201 @@
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import ChristianHolidays, InternationalHolidays, StaticHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SAT_SUN_TO_NEXT_MON,
+)
+
+
+class Tuvalu(ObservedHolidayBase, ChristianHolidays, InternationalHolidays, StaticHolidays):
+    """
+    References:
+      - `Public holidays in Tuvalu <https://en.wikipedia.org/wiki/Public_holidays_in_Tuvalu>`_
+      - `Today's and Upcoming Holidays in Tuvalu <https://www.timeanddate.com/holidays/tuvalu/>`_
+      - `Public Holidays (Amendment) Act 1990 <https://www.paclii.org/cgi-bin/sinodisp/tv/legis/num_act/pha1990243/pha1990243.html>`_
+      - `Public Holidays Act 1 <https://tuvalu-legislation.tv/cms/images/LEGISLATION/PRINCIPAL/1937/1937-0005/1937-0005_1.pdf>`_
+      - `Labour and Employment Relations Act 2017 <https://tuvalu-legislation.tv/cms/images/LEGISLATION/PRINCIPAL/2017/2017-0014/2017-0014_1.pdf>`_
+      - `Public Holidays (Amendment) Act 2018 <https://tuvalu-legislation.tv/cms/images/LEGISLATION/AMENDING/Public%20Holidays%20(Amendment)%20Act/Public%20Holidays%20(Amendment)%20Act%202018/Public%20Holidays%20(Amendment)%20Act%202018.pdf>`_
+      - `Public Holidays (Amendment) Act 2020 <https://tuvalu-legislation.tv/cms/images/LEGISLATION/AMENDING/2020/2020-0013/2020-0013.pdf>`_
+      - `Public Holidays Act 2 <https://tuvalu-legislation.tv/cms/images/LEGISLATION/PRINCIPAL/1937/1937-0005/1937-0005_2.pdf>`_
+      - `Codes for the representation of names of countries and their subdivisions <https://www.iso.org/obp/ui/#iso:code:3166:TV>`_
+      - `TUVALU-NEWS.TV <https://web.archive.org/web/20140915180104/http://www.tuvalu-news.tv/archives/2007/01/island_special_public_holidays.html>`_
+    """
+
+    country = "TV"
+    default_language = "tvl_TV"
+    # %s (estimated).
+    estimated_label = tr("%s (fakaaoga)")
+    # %s (observed).
+    observed_label = tr("%s (fakamatakuga)")
+    # %s (observed, estimated)
+    observed_estimated_label = "%s (fakamatakuga, fakaaoga)"
+    supported_languages = ("en_GB", "en_US", "tvl_TV")
+    subdivisions = (
+        "FUN",  # Funafuti.
+        "NMG",  # Nanumaga.
+        "NMA",  # Nanumea.
+        "NIT",  # Niutao.
+        "NUI",  # Nui.
+        "NKF",  # Nukufetau.
+        "NKL",  # Nukulaelae.
+        "VAI",  # Vaitupu.
+    )
+    subdivisions_aliases = {
+        # Town/Island Councils.
+        "Funafuti": "FUN",
+        "Nanumaga": "NMG",
+        "Nanumea": "NMA",
+        "Niutao": "NIT",
+        "Nui": "NUI",
+        "Nukufetau": "NKF",
+        "Nukulaelae": "NKL",
+        "Vaitupu": "VAI",
+    }
```

Comment:
```suggestion
    subdivisions = (
        "FUN",  # Funafuti.
        "NIT",  # Niutao.
        "NKF",  # Nukufetau.
        "NKL",  # Nukulaelae.
        "NMA",  # Nanumea.
        "NMG",  # Nanumaga.
        "NUI",  # Nui.
        "VAI",  # Vaitupu.
    )
    subdivisions_aliases = {
        # Town/Island Councils.
        "Funafuti": "FUN",
        "Niutao": "NIT",
        "Nukufetau": "NKF",
        "Nukulaelae": "NKL",
        "Nanumea": "NMA",
        "Nanumaga": "NMG",
        "Nui": "NUI",
        "Vaitupu": "VAI",
    }
```
We keep alphabetical order for subdivisions.

Reply:
While `NMG` was changed from "Nanumanga" to "Nanumaga" in 2016, it probably doesn't hurt adding an additional aliases for that since [that's how the island is commonly referred to in the English language](https://books.google.com/ngrams/graph?content=Nanumanga%2C+Nanumaga&year_start=1800&year_end=2022&corpus=en&smoothing=3).
```python
        # Ex-ISO code aliases.
        "Nanumanga": "NMG",
```

Reply:
Sorry for my mistake. When I first read this comment, I misread it and thought the word was the same as mentioned above, eventually ignoring it.